internal/driver/webhtml.go

```diff
@@ -511,6 +615,11 @@ function dotviewer(nodes) {
     return null
   }
 
+  // convert a string to a regexp that matches that string.
+  function quotemeta(str) {
+    return str.replace(/([\\\.?+*\[\](){}])/g, '\\$1')
```

Comment:
I put it on the right so that it matches the text display.

We could easily cap the width of the Name column (I think a CSS max-width property might suffice).

Could you save the html file that contains the long name and place it somewhere accessible so we can play around with different possible layouts?

Also, should we deal with the long Name column issue in a separate change?